pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1336,6 +1336,19 @@ func parseIpspec(ipspec *zconfig.Ipspec,
 				s, err))
 		}
 		config.Subnet = *subnet
+		// set dhcp range in the second half of the subnet(for larger subnets),
+		// set the first address in the dhcp range as gateway
+		prefixLen, _ := subnet.Mask.Size()
+		prefixRange := 0x01 << (32 - prefixLen)
+		start := 1
+		end := prefixRange - 3
```

Comment:
Why - 3 and not -2? Because of the gateway? If so that needs to be documented in a comment.